packages/contentful-migrations/core/0000000000001-create-d-settings-migration.js

```diff
@@ -2,14 +2,15 @@ module.exports = function (migration) {
   const dSettings = migration
     .createContentType('d-settings')
     .name('Data: Settings')
-    .description('')
-    .displayField('name');
+    .description('Global site parameters')
+    .displayField('internal_name');
+
   dSettings
-    .createField('name')
+    .createField('internal_name')
```

Comment:
Just saw that contentful still uses "name" for this as kind of best practice:
https://www.contentful.com/developers/docs/compose/cli-tools/#add-a-new-page-type-to-compose

Are we sure to rename this to `internal_name`?

Reply:
This could lead to a name conflict if we have a content type that has `name` as a field. For example a data set for persons (blog author). In the linked example, they use "name" as the id and "internal name" as the label. Therefore `internal_name` is not entirely wrong. ;-)

Reply:
if this doesn't collide with compose i'm happy with it. I personally also prefer `internal_name`

Reply:
Looks good so far.